packages/apps/job-launcher/server/src/modules/payment/payment.service.ts

```diff
@@ -260,4 +263,29 @@ export class PaymentService {
 
     return totalAmount;
   }
+
+  public async createRefundPayment(dto: PaymentRefundCreateDto) {
+    const rate = await getRate(Currency.USD, TokenId.HMT);
+
+    try {
+        await this.paymentRepository.create({
+            userId: dto.userId,
+            jobId: dto.jobId,
+            source: PaymentSource.BALANCE,
+            type: PaymentType.REFUND,
+            amount: dto.refundAmount,
+            currency: TokenId.HMT,
+            rate: div(1, rate),
+            status: PaymentStatus.SUCCEEDED,
+        });
```

Comment:
If change the order you don't need to do div(1,rate) later
```suggestion
    const rate = await getRate(TokenId.HMT, Currency.USD);

    try {
        await this.paymentRepository.create({
            userId: dto.userId,
            jobId: dto.jobId,
            source: PaymentSource.BALANCE,
            type: PaymentType.REFUND,
            amount: dto.refundAmount,
            currency: TokenId.HMT,
            rate: rate,
            status: PaymentStatus.SUCCEEDED,
        });
```